integr8ly/tutorial-web-app
# INTLY-158: Updated with PF4 react core components.

## Motivation
Added PF4 React Core so we can pull in react components. Updated one of the buttons to PF4 react button to verify that PF4 is being pulled in.

<img width="1328" alt="49825267-018d8e80-fd52-11e8-8ae6-a2f43d161552" src="https://user-images.githubusercontent.com/6126356/49883205-2cceb700-fe00-11e8-9965-2e05c5e76611.png">


## Verification Steps

Build code doing yarn install, yarn build.
Verify that PF4 react is being included by going to the Getting Started page for a tutorial
The button in the top right is now a PF4 react component.

## Checklist:


## Progress
